book/python/api.md

```diff
@@ -444,7 +444,7 @@ A complete `KafkaSink` encoder example that takes a word and sends it to the par
 @wallaroo.encoder
 def kafka_encode(data):
     word = data[:]
```

Comment:
I'm also wondering about the need for this one, since strings are supposed to be immutable? But maybe I'm missing something here.

Reply:
Ah. Makes sense.
I think we should try to come up with a way to hide that from the user... I'll open a discussions-needed issue for that.